### PR TITLE
Fix permissions for world-map

### PR DESCRIPTION
**Pull request by spaprzycki**
As per issue 1855 created short while ago

### CODE REVIEW

**Comment by laf** [comment]
Looks good to me. One of the other guys will give it an actual test hopefully but the query looks fine :)

Thanks for helping out.

**Comment by spaprzycki** [comment]
Hi laf, hopefully it will pass testing... :) Done it as a quick fix for my own needs and works for me, so decided to share it back. I have few other fixes/improvements that I made lately, will 'pull request' what I think may be useful for general use.

Apologies for double-commits.. git is still bit of a mystery for me, hopefully not for long now ;-)

File: html/includes/common/worldmap.inc.php [hunk 1]
```diff
@@ -51,7 +51,25 @@
...
+            ORDER BY `status` ASC, `hostname`";
+}
+else {
+// Normal user - grab devices that user has permissions to
+    $sql = "SELECT `devices`.`device_id` as `device_id`,`hostname`,`os`,`status`,`lat`,`lng`
+            FROM `devices_perms`, `devices`
+            LEFT JOIN `locations` ON `devices`.`location`=`locations`.`location`
+            WHERE `disabled`=0 AND `ignore`=0 AND `lat` != '' AND `lng` != ''
+            AND `devices`.`device_id` = `devices_perms`.`device_id`
+            AND `devices_perms`.`user_id` = '".$_SESSION['user_id']."'
```
Comment (f0o) [inline html/includes/common/worldmap.inc.php]: please use a `?` sign as placeholder and give the `dbFetchRows($sql)` a 2nd param in form of `array($_SESSION['user_id'])`

**Comment by spaprzycki** [comment]
modified as suggested

**Comment by f0o** [comment]
tagging: #1855

**Comment by spaprzycki** [comment]
f0o I think I understand what you asked me to do over IRC yesterday, but since I am working on the fork and then manually create pull requests my checksums for commits are different. I have done what you suggested, but on the corresponding commit from my repo. Can you see if that is ok now and if not what's the best way to solve it..? I'm totally lost :(

**Pull request closed without merging**